crates/graphql/src/schema.rs

```diff
@@ -961,6 +961,31 @@ impl QueryRoot {
 
         rows.pop().map(Into::into)
     }
+
+    fn nfts_by_wallet(&self, context: &AppContext, owner_address: String) -> Vec<Nft> {
```

Comment:
Ideally I'd like this to be a filter in the existing nft query. That maybe be challenging but give it a look and let me know. nfts(owners: []) should be additive to the existing creators filter

Reply:
I have made creators and owners as optional parameters

Reply:
But the nfts query returns all the nfts when no parameter is passed , is that okay or do you want to return empty results when no parameter is passed